csharp/LogicalColumnReader.cs

```diff
@@ -72,6 +72,8 @@ internal static LogicalColumnReader<TElement> Create<TElement>(ColumnReader colu
 
         public abstract TReturn Apply<TReturn>(ILogicalColumnReaderVisitor<TReturn> visitor);
 
+        public abstract long Skip(long numRowsToSkip);
```

Comment:
I know most existing public methods don't have documentation, but it would be good to have a short documentation comment on this method that contrasts it with the physical reader skip method.